bacon/model/Transaction.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  Transaction.swift
+//  bacon
+//
+//  Created by Fabian Terh on 19/3/19.
+//  Copyright © 2019 nus.CS3217. All rights reserved.
+//
+
+import Foundation
+
```

Comment:
should transactiontype enum be in a seperate file?

Reply:
Done!